developer_manual/app/code-signing.rst

```diff
@@ -0,0 +1,144 @@
+============
+Code Signing
+============
+
+.. sectionauthor:: Lukas Reschke <lukas@owncloud.com>
+
+ownCloud supports code signing for the core releases as well as dedicated applications. Code signing gives our users an
```

Comment:
Why do we repeat the whole description here? Isn’t it better to have a single source? Like the base description / intro should be the same.